client/src/pages/Profile.js

```diff
@@ -1,6 +1,14 @@
-import React from "react";
+import React, { useContext } from "react";
 import Page from "../common/Page";
+import { AuthContext } from "../contexts/AuthContext";
 
 export default function Profile() {
-  return <Page></Page>;
+  const { user } = useContext(AuthContext);
+
+  return (
+    <Page>
+      <h2>{user.login}</h2>
+      <img src={`https://github.com/${user.login}.png`} alt={user.login} />
```

Comment:
```suggestion
      <img src={`https://github.com/${user.login}.png`} alt="" />
```
The `alt` is redundant information in this case. See https://webaim.org/techniques/alttext/